File: JavaScript/Faccat/Exercicio18.js
let anoAtual;
let anoDeNascimento;

anoDeNascimento = parseInt(prompt("Digite o ano de nascimento: "));
anoAtual = parseInt(prompt("Digite o ano atual: "));

if (anoAtual - anoDeNascimento >= 16) {
    console.log("Pode votar");
} else {
    console.log("Não pode votar");
}

File: Java/src/Faccat/Exercicio18.java
package Faccat;

import java.util.Scanner;

public class Exercicio18 {
    public static void main(String[] args) {
        int anoAtual;
        int anoDeNascimento;

        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite o ano de nascimento: ");
        anoDeNascimento = scanner.nextInt();
        System.out.println("Digite o ano atual: ");
        anoAtual = scanner.nextInt();

        if (anoAtual - anoDeNascimento >= 16) {
            System.out.println("Pode votar");
        } else {
            System.out.println("Não pode votar");
        }
    }
}
